product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Step 4 - Installing the PGD software
+navTitle: Installing PGD software
+deepToC: true
+---
+
+## Installing the PGD software
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+* Install the PGD packages which include a server specific BDR package and generic PGD proxy and cli packages.
```

Comment:
Seems to me that the specific packages should be listed here in this bullet:  edb-bdr5-<postgresversion>, edb-pgd5-proxy, edb-pgd5-cli

Reply:
Fixed.